DEVELOPMENT.md

```diff
@@ -159,27 +158,25 @@ to start a debugger to which attach from your IDE.
 
 1. Deploy the chart with `ko` and `helm` in debug mode using an image with `delve`:
    ```bash
-   helm install -n default [deployment name] charts/kubearchive \ 
+   helm install kubearchive charts/kubearchive --create-namespace -n kubearchive \ 
    --set apiServer.debug=true \
-   --set-string apiServer.image=$(KO_DEFAULTBASEIMAGE=gcr.io/k8s-skaffold/skaffold-debug-support/go:latest \
-   ko build --disable-optimizations github.com/kubearchive/kubearchive/cmd/api) \
+   --set-string apiServer.imae=$(KO_DEFAULTBASEIMAGE=gcr.io/k8s-skaffold/skaffold-debug-support/go:latest ko build --disable-optimizations github.com/kubearchive/kubearchive/cmd/api) \
```

Comment:
The `--debug` command on `ko build` does something similar, maybe you want to check if it works, so we can reduce this line.

https://ko.build/reference/ko_build/

```suggestion
   --set-string apiServer.imae=$(ko build --debug --disable-optimizations github.com/kubearchive/kubearchive/cmd/api) \
```

Reply:
there's also a typo

```suggestion
   --set-string apiServer.image=$(KO_DEFAULTBASEIMAGE=gcr.io/k8s-skaffold/skaffold-debug-support/go:latest ko build --disable-optimizations github.com/kubearchive/kubearchive/cmd/api) \
```

Reply:
I see that there is a `--debug` flag in the [docs](https://ko.build/features/debugging/) but I tried the latest version of `ko` (v0.15.4) and the flag doesn't seem to be recognized: 

```bash
ko build --debug --disable-optimizations github.com/kubearchive/kubearchive/cmd/api
Error: unknown flag: --debug
```

I'll keep the env variable `KO_DEFAULTBASEIMAGE` with the fix of the typo that skoved is suggesting